src/database/DataTypes/MasterList.js

```diff
@@ -68,16 +72,24 @@ export class MasterList extends Realm.Object {
   }
 
   /**
-   * Find a specifically named order type in the store tag object for this store and masterlist
+   * Find a specifically named order type in the store tag object for this store
+   * and masterlist.
+   *
    * @param  {string}  tags            Current stores tags field
    * @param  {string}  orderTypeName   Name of the orderType to search for
    * @return {object} The matching orderType object
    */
   getOrderType(tags, orderTypeName) {
-    this.getStoreTagObject(tags).orderTypes.reduce(orderType => {
-      if (!(orderType.name === orderTypeName)) return null;
-      return orderType;
-    });
+    return this.getStoreTagObject(tags).orderTypes.filter(
```

Comment:
Array.find() would be slightly neater

Reply:
Fixed c0c9b0c6db00ab473b9009c4aeb0fe45b200e9b1

Reply:
@Chris-Petty agree